### ISSUE TITLE
Error during first make

### ISSUE THREAD

**Issue by Th-eo** [dannye/pokeyellow-gen-II#2]
Followed pret/pokeyellow install instructions. Cloned then ran "git submodule update --init".
```
ERROR: wram.asm(536):
    Constant mustn't be negative: -528
make: *** [Makefile:42: wram.o] Error 5
```
The line it points to refers to this bit of code:
```
wLYOverridesBufferEnd:: ; c900
	ds wPrinterSerialReceived - @ ; line 536
```

**Comment by dannye** [comment]
Long story short, you are using an rgbds v0.3.x and, until I update the
repo, you need to be using rgbds v0.2.5
Sorry. I will try to update my repos soon to be compatible with the most
recent rgbds.

**Comment by RetroKoH** [comment]
Hi, I downgraded to rgbds 0.2.5. Solved that error but got a totally different one...
$ make
rgbasm -h -o wram.o wram.asm
rgblink -n pokeyellow.sym -o pokeyellow.gbc audio.o main.o text.o wram.o
D:\cygwin64\usr\local\bin\rgblink.exe: 'audio.o' is an unsupported version
make: *** [Makefile:47: pokeyellow.gbc] Error 1

DISREGARD: It seems there was an error related to a crash on my end. I've fixed it and it builds.
